packages/ui/tsconfig.json

```diff
@@ -23,7 +23,8 @@
     "paths": {
       "~/*": ["src/*"]
     },
-    "typeRoots": ["../../node_modules/@types", "./src/types"]
+    "typeRoots": ["../../node_modules/@types", "../../node_modules", "./src/types"],
```

Comment:
I added `../../node_modules` since we need to be able to resolve the types from the `vitest` package. I'm pretty sure we can drop the `node_modules/@types` since it should be covered by `node_modules`, but I left it here for simplicity.

Reply:
I wonder if we can we remove typeRoots altogether? 🤔

Reply:
it's dead!